src/Raven.Server/Documents/Patch/ScriptRunnerCache.cs

```diff
@@ -69,15 +69,15 @@ public abstract class Key
             public abstract override int GetHashCode();
         }
 
-        public ScriptRunner.ReturnRun GetScriptRunner(Key key, bool readOnly, out ScriptRunner.SingleRun patchRun)
+        public ScriptRunner.ReturnRun GetScriptRunner(Key key, bool readOnly, out ScriptRunner.SingleRun patchRun, bool ignoreValidationErrors = false)
```

Comment:
I wonder if we woudn't be better by having 2 separate methods:

```
public ScriptRunner.ReturnRun GetScriptRunner(Key key, bool readOnly, out ScriptRunner.SingleRun patchRun)
```

and 

```
public ScriptRunner.ReturnRun GetScriptRunner(Key key, bool readOnly, bool ignoreValidationErrors  out ScriptRunner.SingleRun patchRun)
```
It's a bit strange that `out` isn't the last

Reply:
Done.